README.md

```diff
@@ -1,41 +1,5 @@
 <a name="readme-top"></a>
```

Comment:
You can also add an image like your app logo, to make the README file look more professional🤔